test/e2e/scheduling/priorities.go

```diff
@@ -399,3 +443,27 @@ func addRandomTaitToNode(cs clientset.Interface, nodeName string) *v1.Taint {
 	framework.ExpectNodeHasTaint(cs, nodeName, &testTaint)
 	return &testTaint
 }
+
+func UpdateMemoryOfNode(c clientset.Interface, nodeName string, value int64) error {
```

Comment:
nit: s/value/memory/

Also, please add a comment for the function and perhaps make it private (change to lower case).